tests/integration/tier1/convert-offline-systems/run_conversion.py

```diff
@@ -10,7 +10,7 @@ def test_convert_offline_systems(shell, convert2rhel):
     # The CentOS8 Extras repo url is unreachable due to offline system setup.
     # The repoquery returns an error, thus we need to disable this repository.
     if "centos-8" in platform.platform():
-        shell("yum-config-manager --disable extras --disable epel-modular")
+        shell("yum-config-manager --disable extras --disable epel-modular --disable appstream")
```

Comment:
@r0x0d, @kokesak, @danmyway, why is appstream being disabled and not some other repos like baseos?

Reply:
@bocekm this was just me playing with the test to see what happens. 
The issue I'm hitting here is this:
```
20:29:42             out: [05/12/2022 16:29:42] TASK - [Prepare: Checking if --enablerepo repositories are accessible] ****
20:29:42             out: Skipping the check of repositories due to the use of RHSM for the conversion.
20:29:42             out: 
20:29:42             out: [05/12/2022 16:29:42] TASK - [Prepare: Checking if the packages are up-to-date] *****************
20:29:42             out: Errors during downloading metadata for repository 'appstream':
20:29:42             out:   - Curl error (7): Couldn't connect to server for https://vault.centos.org/centos/8.5.2111/AppStream/x86_64/os/repodata/repomd.xml [Failed to connect to vault.centos.org port 443: Connection refused]
20:29:42             out: [05/12/2022 16:29:42] DEBUG - Traceback (most recent call last):
20:29:42             out:   File "/usr/lib/python3.6/site-packages/dnf/repo.py", line 574, in load
20:29:42             out:     ret = self._repo.load()
20:29:42             out:   File "/usr/lib64/python3.6/site-packages/libdnf/repo.py", line 397, in load
20:29:42             out:     return _repo.Repo_load(self)
20:29:42             out: libdnf._error.Error: Failed to download metadata for repo 'appstream': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
20:29:42             out: 
20:29:42             out: During handling of the above exception, another exception occurred:
20:29:42             out: 
20:29:42             out: Traceback (most recent call last):
20:29:42             out:   File "/usr/lib/python3.6/site-packages/convert2rhel/main.py", line 91, in main
20:29:42             out:     checks.perform_pre_checks()
20:29:42             out:   File "/usr/lib/python3.6/site-packages/convert2rhel/checks.py", line 61, in perform_pre_checks
20:29:42             out:     check_package_updates()
20:29:42             out:   File "/usr/lib/python3.6/site-packages/convert2rhel/checks.py", line 476, in check_package_updates
20:29:42             out:     packages_to_update = get_total_packages_to_update(reposdir=reposdir)
20:29:42             out:   File "/usr/lib/python3.6/site-packages/convert2rhel/pkghandler.py", line 1036, in get_total_packages_to_update
20:29:42             out:     packages = _get_packages_to_update_dnf(reposdir=reposdir)
20:29:42             out:   File "/usr/lib/python3.6/site-packages/convert2rhel/pkghandler.py", line 1091, in _get_packages_to_update_dnf
20:29:42             out:     base.fill_sack()
20:29:42             out:   File "/usr/lib/python3.6/site-packages/dnf/base.py", line 399, in fill_sack
20:29:42             out:     self._add_repo_to_sack(r)
20:29:42             out:   File "/usr/lib/python3.6/site-packages/dnf/base.py", line 139, in _add_repo_to_sack
20:29:42             out:     repo.load()
20:29:42             out:   File "/usr/lib/python3.6/site-packages/dnf/repo.py", line 581, in load
20:29:42             out:     raise dnf.exceptions.RepoError(str(e))
20:29:42             out: dnf.exceptions.RepoError: Failed to download metadata for repo 'appstream': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

This happens on C8.5. On C8.4 it has problem with "extras" repo. The system is "offline" so it cannot reach those repos. However it's strange that even though I disabled those repos it still tries to download from them

Reply:
full log here:
C8.5: http://artifacts.osci.redhat.com/testing-farm/801e69bd-b1af-440a-9328-644ba6d4024e/work-convert_offline_systems2BBlv3/log.txt
C8.4: http://artifacts.osci.redhat.com/testing-farm/0ef7b1f9-f411-4856-a320-7936105b8d24/work-convert_offline_systemsRcB199/log.txt

Reply:
```
Errors during downloading metadata for repository 'appstream':
 - Curl error (7): Couldn't connect to server for https://vault.centos.org/centos/8.5.2111/AppStream/x86_64/os/repodata/repomd.xml [Failed to connect to vault.centos.org port 443: Connection refused]
```
The URL is accessible now so I bet it was some kind of intermittent vault outage.

Reply:
It is accessible just not on the testing machine. This is the "offline conversion" test. The machine has access only to our internal satellite server. This problem is only on CentoOS 8.5/8.4 as far as I looked at the tests, should the repositories be available on the satellite or how should I approach this? Remove the unaccessible repos from the system before the conversion? This seems a bit hackisch to me :D

Reply:
Ah, right, this is the offline test. Then I think disabling all the repositories that require the internet connection is perfectly fine. Or removing the repofiles that define such repositories. That's what expect customer would do. Why would they keep repositories they don't have access to enabled..